src/Commands/core/LocaleCommands.php

```diff
@@ -208,6 +209,78 @@ public function exportValidate(CommandData $commandData): void
         }
     }
 
+    /**
+     * Imports multiple translations from the defined directory.
+     *
+     * @throws \Exception
+     */
+    #[CLI\Command(name: self::IMPORT_ALL, aliases: ['locale-import-all', 'locale:import:all'])]
+    #[CLI\Argument(name: 'directory', description: 'The path to directory with .po files to import.')]
+    #[CLI\Option(name: 'type', description: 'String types to include, defaults to <info>not-customized</info>. Recognized values: <info>not-customized</info>, <info>customized</info>')]
+    #[CLI\Option(name: 'override', description: 'Whether and how imported strings will override existing translations. Defaults to the Import behavior configured in the admin interface. Recognized values: <info>none</info>, <info>customized</info>, <info>not-customized</info>, <info>all</info>')]
+    #[CLI\Usage(name: 'drush locale:import-all /var/www/translations', description: 'Import all custom translations from the defined directory.')]
+    #[CLI\Usage(name: 'drush locale:import-all --types=customized --override=all', description: 'Import all custom translations from the defined directory and override any existing translation.')]
```

Comment:
No directory is provided in the example. /var/www/translations/custom

Reply:
fixed